glpi-project/glpi
# Fix/9.5/ate 20191216

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Some minor fixes:
- add is_recursive in external event to ease usage
- lowercase rrule repetition (some caldav clients send in uppercase, other in lower)
- add a flag to choose what's state should have an external event when added from caldav client (it may be useful to force checking planning conflict in GLPI).
